src/main/joyride/nrepl.cljs

```diff
@@ -182,51 +184,105 @@
            (debug "Connection lost")
            (debug "Connection closed")))))
 
-(defn start-server
+(defn port-file-uri [root-uri]
+  (vscode/Uri.file
+   (path/join root-uri ".joyride" ".nrepl-port")))
+
+(defn remove-port-file [^js path]
+  (let [uri (port-file-uri path)]
+    (-> uri
+     vscode/workspace.fs.stat
+     (p/then (fn [stat]
+               (when stat
+                 (vscode/workspace.fs.delete uri))))
+     (p/catch (fn [e]
```

Comment:
Why catch and log here instead of not breaking the exception flow for users to catch the error?

Reply:
My thinking here was that we have no API for stopping the server. But, I guess someone could execute the command...